situp-plugins/apmtracesource/build.gradle

```diff
@@ -7,6 +7,20 @@ dependencies {
     compile project(':situp-api')
     compile project(':situp-plugins:common')
     implementation 'io.netty:netty-all:4.1.51.Final'
+    implementation 'io.opentelemetry:opentelemetry-proto:0.6.0'
+    implementation 'com.google.protobuf:protobuf-java-util:3.11.0'
     implementation "com.fasterxml.jackson.core:jackson-databind:${versionMap.jackson_databind}"
     implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${versionMap.jackson_dataformat_yaml}"
+    testImplementation 'org.assertj:assertj-core:3.6.2'
```

Comment:
Looks like we are getting this dependency only for assertThat? Can we not just use ```org.junit.Assert``` itself?

Reply:
I found assertThat to be more descriptive and readable than org.junit.Assert.

Reply:
u can use assertThat from hamcrest which is what we are doing in other packages. not a blocker but probably in ur next PR if you can check to use hamcrest, it will be uniform.